include/ppx/application.h

```diff
@@ -559,6 +563,23 @@ class Application
     // Initializes standard knobs
     void InitStandardKnobs();
 
+    // List gpus
+    void ListGPUs() const;
+
+    // Process events which could change the Running status of the application
+    void ProcessEvents();
+
+    // Render the frame, handles both XR and non-XR cases
+    void FrameRender();
```

Comment:
nit: can we rename this to RenderFrame()? imo makes it clearer to put verb first, but feel free to ignore if you disagree

Reply:
renamed the function as suggested